<launch>

  <!-- Basic data -->
  <arg name="namespace"                 default="drone1" />
  <arg name="drone_id"                           default="1" />
  <arg name="my_stack_directory"                 default="$(env AEROSTACK_STACK)" />
 
  <!-- topics & services-->
  <arg name="add_belief_srv"                     default="add_belief" />
 
  <group ns="$(arg namespace)">

    <!-- Basic data -->
    <param name="~namespace"     value="$(arg namespace)"    type="str" />
    <param name="~drone_id"               value="$(arg drone_id)"              type="str" />
    <param name="~my_stack_directory"     value="$(arg my_stack_directory)"    type="str" />
   

    <!-- topics & services -->
    <param name="~add_belief_srv"         value="$(arg add_belief_srv)"        type="str" />
   

    <node name="behavior_inform_operator" pkg="behavior_inform_operator"  type="behavior_inform_operator"  output="screen"/>
    <node name="behavior_request_operator_assistance" pkg="behavior_request_operator_assistance"  type="behavior_request_operator_assistance"  output="screen"/>

    <test test-name="behavior_inform_operator_test"  pkg="behavior_inform_operator_assistance" type="behavior_inform_operator_test"/>
    <test test-name="behavior_request_operator_assistance_test"  pkg="behavior_request_operator_assistance" type="behavior_request_operator_assistance_test"/>

  </group>

</launch>
